src/views/TechnicalAssistance/TaskList.tsx

```diff
@@ -196,7 +196,7 @@ function TaskList() {
                       </Grid>
                     )}
 
-                  {data?.trbRequest.feedback.length !== 0 && (
+                  {data?.trbRequest.feedback.length > 0 && (
```

Comment:
When would this be different from `!==`? If `data.trbRequest` isn't defined? (or is this just for clarity, and the real change is the consultPrepStatus change below?